### PR TITLE
Adds new integration [albaintor/homeassistant_electrolux_status]

### PR DESCRIPTION
**Pull request by albaintor**
Removed the old integration which has been deleted and added the new forked one

## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: https://github.com/albaintor/homeassistant_electrolux_status/releases/tag/v1.0.4
Link to successful HACS action (without the `ignore` key): https://github.com/albaintor/homeassistant_electrolux_status/actions/runs/7798416502
Link to successful hassfest action (if integration): https://github.com/albaintor/homeassistant_electrolux_status/actions/runs/7798416499

### CODE REVIEW

**Comment by tjorim** [comment]
Don't add and remove an integration in 1 PR. You also didn't sort the list alphabetically.

**Comment by albaintor** [comment]
Hi,
I have applied the requested modifications

**Comment by albaintor** [comment]
Hi,
do you have any idea when this patch will be approved or do I have to do something on my side ?
Thank you

File: integration [hunk 1]
```diff
@@ -1001,5 +1002,5 @@
   "zachowj/hass-node-red",
   "zeronounours/HA-custom-component-energy-meter",
   "zigul/HomeAssistant-CEZdistribuce",
-  "ZsBT/hass-w1000-portal"
-]
\ No newline at end of file
+  "ZsBT/hass-w1000-portal",
```
Comment (ludeeus) [inline integration:1005]:
```suggestion
  "ZsBT/hass-w1000-portal"
```

**Pull request merged**